pool/web/view/page/page_layout.ml

```diff
@@ -257,7 +259,7 @@ let create_error_layout children =
   in
   let content = main_tag [ children ] in
   html
-    (head page_title ([ charset; viewport; favicon ] @ global_stylesheets))
```

Comment:
Why did you remove the `favicon` from the error layour?

Reply:
It is the UZH logo. I don't think we should display that on all tenant pages if an error occurs.

Reply:
Ah I see, ok